### PR TITLE
Adding Handlers for security.txt and humans.txt to robotstxt module

### PR DESCRIPTION
**Pull request by jadhub**
see https://securitytxt.org/ and http://humanstxt.org/ for further info on what these files are about.

Configurable, per default deactivated

### CODE REVIEW

**Comment by jadhub** [comment]
> Can you add web.RoutesModule (Routes method) to Module, to allow this to be used as a normal flamingo module too?

Please review changes, standing by for further changes.

File: core/robotstxt/interfaces/file_controller.go [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+	"context"
+	"flamingo.me/flamingo/v3/framework/flamingo"
+	"flamingo.me/flamingo/v3/framework/web"
+	"io/ioutil"
+	"net/http"
+)
+
+type (
+	// FileControllerInterface is the callback HTTP action provider
+	FileControllerInterface interface {
```
Comment (bastianccm) [inline core/robotstxt/interfaces/file_controller.go:14]: Do we need the interface here?

File: core/robotstxt/interfaces/file_controller.go [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+	if err != nil {
+		d.logger.WithContext(ctx).Error(err)
+
+		d.responder.ServerError(err)
+	}
+
+	return d.responder.HTTP(http.StatusOK, bytes.NewReader(fileContent))
+}
+
+func (d *DefaultFileControllerInterface) readFile(filePath string) ([]byte, error) {
```
Comment (bastianccm) [inline core/robotstxt/interfaces/file_controller.go:79]: I think it's not necessary to wrap ioutil.ReadFile.

File: core/robotstxt/module.go [hunk 3]
```diff
@@ -1,36 +1,78 @@
...
+	routes struct {
+		securityTxtActivated bool
+		humansTxtActivated   bool
+		files                interfaces.FileControllerInterface
 	}
 )
 
 // Configure DI
 func (m *Module) Configure(injector *dingo.Injector) {
-	if m.DefaultMux != nil {
```
Comment (bastianccm) [inline core/robotstxt/module.go:19]: I think, for backwards compatibility, we still need the mechanism to support the DefaultMux here

File: core/robotstxt/module.go [hunk 4]
```diff
@@ -1,36 +1,78 @@
...
+) {
+	r.securityTxtActivated = config.SecurityTxtActivated
+	r.humansTxtActivated = config.HumansTxtActivated
+	r.files = files
+}
+
+// Routes module
+func (r *routes) Routes(registry *web.RouterRegistry) {
+	registry.HandleGet("robotstxt.robotstxt", r.files.GetRobotsTxt)
+	_, err := registry.Route("/robots.txt", "robotstxt.robotstxt")
```
Comment (bastianccm) [inline core/robotstxt/module.go:58]: Use `registry.MustRoute` here instead

**Comment by tessig** [comment]
@jadhub Do you want to go on on this or mind if someone else take over here?

**Review by carstendietrich — APPROVED** [review]
Looks good to me

**Comment by tessig** [comment]
@bastianccm please re-review the changes

**Review by bastianccm — APPROVED** [review]

**Pull request merged**